Sources/Sentry/Public/SentryOptions.h

```diff
@@ -198,6 +198,12 @@ NS_SWIFT_NAME(Options)
  * Default value is <code>NO</code>
  */
 @property (nonatomic, assign) BOOL attachScreenshot;
+
+/**
+ * When enabled, the SDK start transactions for UI Events like Buttons clicks, Switch toggle,
+ * Scrollview interaction and others.
```

Comment:
I think we should define all UI events and remove others.
```suggestion
 * When enabled, the SDK creates transactions for UI events like buttons clicks, switch toggles,
 * and scroll view interactions.
```

Reply:
Can you elaborate? I don't think I get it.